pkgs/applications/graphics/ufraw/default.nix

```diff
@@ -60,7 +60,7 @@ stdenv.mkDerivation {
   ] ++ stdenv.lib.optional withGimpPlugin "--with-gimp";
 
   postInstall = stdenv.lib.optionalString withGimpPlugin ''
-    moveToOutput "lib/gimp" "$gimpPlugin"
+    moveToOutput ${gimp.targetPluginDir} "$gimpPlugin"
```

Comment:
Could you move this to a separate commit?

Reply:
sure